specification/compute/resource-manager/Microsoft.Compute/stable/2022-04-04/CloudServiceRP/cloudService.json

```diff
@@ -2248,6 +2282,25 @@
         }
       }
     },
+    "SystemData": {
+      "description": "The system meta data relating to this resource.",
+      "type": "object",
+      "properties": {
+        "createdAt": {
+          "format": "date-time",
+          "description": "Specifies the time in UTC at which the Cloud Service (extended support) resource was created. <br />Minimum api-version: 2022-04-04.",
+          "type": "string",
+          "readOnly": true
+        },
+        "lastModifiedAt": {
+          "format": "date-time",
+          "description": "Specifies the time in UTC at which the Cloud Service (extended support) resource was last modified. <br />Minimum api-version: 2022-04-04.",
+          "type": "string",
+          "readOnly": true
+        }
+      },
+      "readOnly": true
+    },
```

Comment:
I notice that this SystemData is only a subset of the systemData defined in common-types.
Could we use the systemData in common-types instead? Or move this to the common.json?

Reply:
good point thank you

Reply:
@arpitkh12  could you move this object to common.json and refer to it instead?